### PR TITLE
fix: hmr, file resolution warnings

### PR DESCRIPTION
**Pull request by acao**
- hmr bug was loading minified version in dev server index.html
- file resolution warnings from .d.ts and map files.
  - we will want those for ts-loader soon!
- analyzer path for deploy previews at /analyzer

### CODE REVIEW

File: packages/graphiql/resources/webpack.config.js [hunk 1]
```diff
@@ -49,12 +50,14 @@ const resultConfig = {
         test: /\.mjs$/,
         use: [],
         include: /node_modules/,
+        exclude: /\.(ts|d\.ts|d.\ts\.map)$/,
```
Comment (ryan-m-walker) [inline packages/graphiql/resources/webpack.config.js]:
```suggestion
        exclude: /\.(ts|d\.ts|d\.ts\.map)$/,
```

File: packages/graphiql/resources/webpack.config.js [hunk 2]
```diff
@@ -49,12 +50,14 @@ const resultConfig = {
...
       },
       // i think we need to add another rule for
       // codemirror-graphql esm.js files to load
       {
         test: /\.(js|jsx)$/,
         use: [{ loader: 'babel-loader' }],
+        exclude: /\.(ts|d\.ts|d.\ts\.map)$/,
```
Comment (ryan-m-walker) [inline packages/graphiql/resources/webpack.config.js]:
```suggestion
        exclude: /\.(ts|d\.ts|d\.ts\.map)$/,
```

Comment (acao) [reply packages/graphiql/resources/webpack.config.js]: doh

**Comment by ryan-m-walker** [comment]
don't forget to swap out this `isDev` for `isHMR` too:

https://github.com/graphql/graphiql/pull/1097/files#diff-a221a80b5413505f6c7c59b760bb3860R68

**Pull request merged**